### PR TITLE
Add lithology chart and tables with well log data

### PR DESCRIPTION
**Pull request by danielnaab**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Lithology / Well log
-----------
A separate chart was added for lithology, with domain lines mapping the extents from the main chart to the lithology chart. Tables replicating what's displayed on the current well log tab were added.

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: assets/src/scripts/components/graph/view/lithology.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    '#72655C',
+    '#CDE569',
+    '#C6BBB7'
+];
+
+
+export default function (elem, {lithology}, container) {
+    container = container || elem
+        .append('g');
+            //.style('opacity', '.5');
```
Comment (ayan-usgs) [inline assets/src/scripts/components/graph/view/lithology.js]: Is there a reason this is commented out?

Comment (danielnaab) [reply assets/src/scripts/components/graph/view/lithology.js]: It doesn't look good - I'll remove

File: assets/src/scripts/components/graph/view/lithology.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            .style('opacity', '0')
+            .remove();
+
+    const newRects = rects.enter()
+        .append('rect')
+            .attr('x', datum => datum.x)
+            .attr('y', datum => datum.y)
+            .attr('width', datum => datum.width)
+            .attr('height', datum => datum.height)
+            .attr('fill', (_, index) => LITHOLOGY_COLORS[index % LITHOLOGY_COLORS.length]);
```
Comment (ayan-usgs) [inline assets/src/scripts/components/graph/view/lithology.js]: I like this... it a really cunning way to select from four colors.

File: assets/src/scripts/services/state/water-levels.spec.js [hunk 3]
```diff
@@ -2,12 +2,12 @@ import { combineReducers, createStore } from 'redux';
 import configureStore from 'redux-mock-store';
 import { default as thunk } from 'redux-thunk';
 
-import cache from './cache';
-import { getWaterLevels, retrieveWaterLevels, setWaterLevels, WATER_LEVELS_SET } from './cache';
+import cache from './water-levels';
```
Comment (ayan-usgs) [inline assets/src/scripts/services/state/water-levels.spec.js]: Could `cache` be imported with the stuff in the line below?

File: assets/src/scripts/services/state/well-log.spec.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { combineReducers, createStore } from 'redux';
+
+import cache from './well-log';
```
Comment (ayan-usgs) [inline assets/src/scripts/services/state/well-log.spec.js]: Same question as the other `cache`.

File: assets/src/scripts/services/state/water-levels.spec.js (hunk 3 above)
Comment (danielnaab) [reply assets/src/scripts/services/state/water-levels.spec.js]: Yes, we could do: ```import { default as waterLevels } from './water-levels';```

**Pull request merged**